cql3/statements/alter_table_statement.cc

```diff
@@ -8,13 +8,17 @@
  * SPDX-License-Identifier: (AGPL-3.0-or-later and Apache-2.0)
  */
 
+#include <cstddef>
```

Comment:
Is this header needed for something here?

Reply:
no, removed